py/fiberassign/assign.py

```diff
@@ -336,30 +336,33 @@ def write_assignment_fits_tile(asgn, fulltarget, overwrite, params):
                     # position in the hardware model.  Used this fixed fiber location.
                     xy = hw.thetaphi_to_xy(
                         hw.loc_pos_curved_mm[loc],
-                        hw.loc_theta_pos[loc],
-                        hw.loc_phi_pos[loc],
+                        hw.loc_theta_pos[loc] + hw.loc_theta_offset[loc],
+                        hw.loc_phi_pos  [loc] + hw.loc_phi_offset  [loc],
                         hw.loc_theta_arm[loc],
                         hw.loc_phi_arm[loc],
                         hw.loc_theta_offset[loc],
                         hw.loc_phi_offset[loc],
                         hw.loc_theta_min[loc],
                         hw.loc_phi_min[loc],
                         hw.loc_theta_max[loc],
-                        hw.loc_phi_max[loc]
+                        hw.loc_phi_max[loc],
+                        ignore_range=True
                     )
                     empty_x[iloc] = xy[0]
                     empty_y[iloc] = xy[1]
                 else:
```

Comment:
Parking an unassigned positioner.  Here the theta/phi offsets were already being applied.